docs/statsd.rst

```diff
@@ -13,7 +13,9 @@ Configuration
 Statsd can be configured by the STATSD_DSN envvar, patterned after the SENTRY_DSN.
 This combines all statsd config into a single DSN url.
 
-e.g.::
+e.g.
```

Comment:
I'd collapse this into the previous paragraph, or alternatively use 'For example::' instead